src/batched/KokkosBatched_SolveUTV_TeamVector_Internal.hpp

```diff
@@ -142,39 +143,45 @@ namespace KokkosBatched {
 		   one,
 		   T, ts0, ts1,
 		   W, ws0, ws1);
+	member.team_barrier();
```

Comment:
possibly not needed since Trsm should always end on a diagonal matrix entry, the calculation of which has a team_barrier(). Maybe you could verify?